src/components/molecules/Navbar.tsx

```diff
@@ -0,0 +1,41 @@
+import { Box, Button } from "@mui/material";
+
+import NavList from "./NavList";
+import Logo from "../atoms/Logo";
+import React from "react";
+import { colors } from "../../theme/colors";
+
+const STYLES = {
+  nav: {
+    backgroundColor: "#084b3c",
```

Comment:
move these colors to `colors.ts` file, if not a part of theme and then import the definition here.

```
backgroundColor: "primary.main"
```suggestion
    backgroundColor: "#084b3c",
```
```